lib/guardian/post_guardian.rb

```diff
@@ -58,7 +58,19 @@ def post_can_act?(post, action_key, opts: {}, can_see_post: nil)
 
         if action_key == :notify_user &&
              !@user.in_any_groups?(SiteSetting.personal_message_enabled_groups_map)
-          return false
+          # The modifier below is used here to add additional permissions for notifying users.
+          # In core the only method of notifying a user is personal messages so we check if the
+          # user can PM. Plugins can extend the behavior of users are notifier via `notify_user`
+          # post action, and this allows extension for that use case.
```

Comment:
I think there might be a typo here? This doesn't make sense to me 🤔 

> Plugins can extend the behavior of users are notifier via `notify_user` post action, and this allows extension for that use case.

Reply:
yep, fixed